keadm/cmd/keadm/app/cmd/debug/check.go

```diff
@@ -285,7 +285,7 @@ func CheckNetWork(IP string, timeout int, cloudhubServer string, edgecoreServer
 	if config != "" {
 		edgeConfig, err := util.ParseEdgecoreConfig(config)
 		if err != nil {
-			err = fmt.Errorf("parse Edgecore config failed")
+			return fmt.Errorf("parse Edgecore config failed")
 		} else {
```

Comment:
The else here can be deleted :)

Reply:
I think the else can't be deleted.The variable cloudhubServer need to be set a default value if it's not assigned.